package.json

```diff
@@ -50,7 +50,6 @@
         "lint:versions": "node ./scripts/lint-versions.js",
         "new-package": "cd projects/templates && plop",
         "postcustom-element-json": "node ./tasks/run-check-cem.js",
-        "postdocs:analyze": "node ./scripts/add-custom-properties.js --src=\"projects/documentation/custom-elements.json\"",
```

Comment:
This is reading in spectrum-vars.json files from the packages in this repo and those files are legacy and no longer created or committed.